pkg/sql/testdata/fk

```diff
@@ -510,3 +510,25 @@ DROP TABLE otherdb.othertable
 
 statement ok
 DROP TABLE otherdb.othertable, crossdb
+
+statement ok
+CREATE TABLE modules (
+  id          BIGSERIAL    NOT NULL PRIMARY KEY,
+  name        VARCHAR(128) NOT NULL UNIQUE
+);
+
+statement ok
+CREATE TABLE domains (
+  id             BIGSERIAL    NOT NULL PRIMARY KEY,
+  domain         VARCHAR(128) NOT NULL UNIQUE
+);
+
+statement ok
+CREATE TABLE domain_modules (
+  id         BIGSERIAL    NOT NULL PRIMARY KEY,
+  domain_id  BIGINT       NOT NULL,
+  module_id  BIGINT       NOT NULL,
+  CONSTRAINT domain_modules_domain_id_fk FOREIGN KEY (domain_id) REFERENCES domains (id),
+  CONSTRAINT domain_modules_module_id_fk FOREIGN KEY (module_id) REFERENCES modules (id),
+  CONSTRAINT domain_modules_uq UNIQUE (domain_id, module_id)
+);
```

Comment:
Do these tests really test the prefix key case. I thought the FK had to reference only a subset of the primary key?